### ISSUE TITLE
[Bug]: Enemy Barrier amount is not saved, and resets to zero upon loading a save

### ISSUE THREAD

**Issue by Flurryous** [MelvorIdle/melvoridle.github.io#4492]
### Describe the bug

For some reason, offline progression can give you a free kill on a barrier monster even when you have no sources of barrier damage. 

### Reproduction Steps

1. Equip only a weapon, such as a bronze dagger or an agile wings rapier. 
2. Fight one of the barrier mobs in the Eye-conic Cave. 
3. Log out.
4. Wait a few seconds and log back in.
5. Watch as offline progression calculates that you managed to kill a single mob.

### Expected behaviour

Without any source of barrier damage, you shouldn't be able to get a barrier monster kill at any point. 

### Save Link - Required if issue occurs with a save

https://melvoridle.com/save/JKOxYOg1

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [X] Steam
- [ ] Epic Games
- [ ] Mobile App (iOS)
- [ ] Mobile App (Android)
- [ ] Mobile App (Android Beta)
- [ ] Other (Please Specify)

### Which expansions are purchased and active at the time of the bug?

- [X] Throne of the Herald
- [X] Atlas of Discovery
- [X] Into the Abyss

### Which version of the game are you experiencing this bug on?

v1.3

### Game Subversion

?11560

### Are you using any mods?

Yes

**Comment by MelvorIdle** [comment]
I am unable to reproduce this bug. There's a chance you have very small source of Barrier damage somewhere, either via a Potion or a chance to apply some DoT that seems to be getting through the Barrier.

Can you please provide a save for us to look at that causes the issue?

**Comment by Flurryous** [comment]
> I am unable to reproduce this bug. There's a chance you have very small source of Barrier damage somewhere, either via a Potion or a chance to apply some DoT that seems to be getting through the Barrier.
> 
> Can you please provide a save for us to look at that causes the issue?

Yes, I have edited in a save file (https://melvoridle.com/save/JKOxYOg1). The character is currently running the voodoo trinket necklace and infernal claw only against the blind ghost.

I tested with a weapon that gave me 0% accuracy before that, and relogging while fighting the blind ghost shows that the barrier has been taken down somehow. It seems like the game forgets to load in the monster's barrier value when you log in?
